### PR TITLE
Create writing-an-mvp.md

### PR DESCRIPTION
**Pull request by justinegeffen**
New sub-section for the voice, tone, style guidelines

### CODE REVIEW

File: operations/operations/publishing/publishing-guidelines/writing-an-mvp.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Writing an MVP
+
+Our technical documentation lives at [docs.mattermost.com](https://docs.mattermost.com), primarily written using ReStructured Text, using Sphinx and Write The Docs as our build and theme tools.
```
Comment (amyblais) [inline operations/operations/publishing/publishing-guidelines/writing-an-mvp.md]: I would say maybe say "admin documentation" or potentially both admin and technical.

File: operations/operations/publishing/publishing-guidelines/writing-an-mvp.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Writing an MVP
+
+Our technical documentation lives at [docs.mattermost.com](https://docs.mattermost.com), primarily written using ReStructured Text, using Sphinx and Write The Docs as our build and theme tools. 
+
+If your PR is for a new feature, major update to an existing feature, or a fix that impacts the current documentation it's recommended that you include documentation around the change. This is helpful in providing context for reviewers, and also expedites the content being added to the technical documentation.
```
Comment (amyblais) [inline operations/operations/publishing/publishing-guidelines/writing-an-mvp.md]:
```suggestion
If your PR is for a new feature, major update to an existing feature, or a fix that impacts the current documentation, it's recommended that you include documentation around the change. This is helpful in providing context for reviewers, and also expedites the content being added to the technical documentation. 
```

File: operations/operations/publishing/publishing-guidelines/writing-an-mvp.md (hunk 1 above)
Comment (justinegeffen) [reply operations/operations/publishing/publishing-guidelines/writing-an-mvp.md]: Good point. What about "admin, user, and technical documentation"?

Comment (amyblais) [reply operations/operations/publishing/publishing-guidelines/writing-an-mvp.md]: That sounds good.

File: operations/operations/publishing/publishing-guidelines/writing-an-mvp.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Writing and submitting documentation with your PR
+
+Our user, admin, and technical documentation lives at [docs.mattermost.com](https://docs.mattermost.com), primarily written using ReStructured Text, using Sphinx and Write The Docs as our build and theme tools. 
+
+If your PR is for a new feature, major update to an existing feature, or a fix that impacts the current documentation, it's recommended that you include documentation around the change. This is helpful in providing context for reviewers, and also expedites the content being added to the technical documentation.
```
Comment (amyblais) [inline operations/operations/publishing/publishing-guidelines/writing-an-mvp.md]: Maybe in places where we refer to "technical documentation" we say docs.mattermost.com instead. I'm 0/5 and I'm not sure what is the best way to refer to the docs site.

On https://developers.mattermost.com/ at the top far-right we refer to it as "Admin docs".

**Review by jasonblais — APPROVED** [review]
Excellent, great work Justine :+1:

File: operations/operations/publishing/publishing-guidelines/writing-an-mvp.md (hunk 3 above)
Comment (justinegeffen) [reply operations/operations/publishing/publishing-guidelines/writing-an-mvp.md]: Done. :)

**Review by amyblais — APPROVED** [review]

**Pull request merged**